article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -153,13 +153,13 @@ object ArticlePicker {
       ("isNotAMP", ArticlePageChecks.isNotAMP(request)),
       ("isNotPaidContent", ArticlePageChecks.isNotPaidContent(page)),
       ("isSupportedTone", ArticlePageChecks.isSupportedTone(page)),
-      ("isNotBlackListed", ArticlePageChecks.isNotBlackListed(page)),
+      ("isNotDenyListed", ArticlePageChecks.isNotDenyListed(page)),
```

Comment:
Super minor but I wonder if `isNotInDenyList` would read slightly better here? It's also perhaps more consistent with `isInAllowList`.

Reply:
👍

Reply:
how about `blocklist`? sounds more natural to me (and probably actually clearer than `blacklist` too)

Reply:
+1